### PR TITLE
explicitly output hex and dec values

### PR DESCRIPTION
**Pull request by usha1830**

### CODE REVIEW

File: backends/dpdk/spec.cpp [hunk 1]
```diff
@@ -455,16 +452,17 @@ std::ostream& IR::DpdkLearner::toSpec(std::ostream& out) const {
     }
     out << std::endl;
     if (auto size = properties->getProperty("size")) {
-        out << "\tsize " << DPDK::toStr(size->value) << "" << std::endl;
+        out << "\tsize 0x" << std::hex << DPDK::toStr(size->value) << "" << std::endl;
```
Comment (psivanup) [inline backends/dpdk/spec.cpp]: Hope std::hex is not required here as the next operand is a string.

**Review by psivanup — COMMENTED** [review]
Please check the comment.

File: backends/dpdk/spec.cpp (hunk 1 above)
Comment (usha1830) [reply backends/dpdk/spec.cpp]: Yes. Thanks for pointing out. Will update

Comment (usha1830) [reply backends/dpdk/spec.cpp]: updated

**Review by psivanup — APPROVED** [review]
LGTM

**Review by github-sajan — APPROVED** [review]
Looks fine

**Comment by usha1830** [comment]
Merged to upstream main

**Pull request closed without merging**